.changeset/slow-flies-hear.md

```diff
@@ -0,0 +1,7 @@
+---
+"@rocket.chat/meteor": minor
+"@rocket.chat/core-typings": minor
+"@rocket.chat/model-typings": minor
+---
+
+Replaces Livechat Visitors by Contacts on workspaces' MAC count
```

Comment:
Think we should get a more product-ish description here as this may affect pricing (even if its for good)